derive_builder_core/src/setter.rs

```diff
@@ -151,7 +154,26 @@ impl<'a> ToTokens for Setter<'a> {
                         let mut new = #self_into_return_ty;
                         new.#field_ident = ::derive_builder::export::core::option::Option::Some(converted);
                         Ok(new)
-                }));
+                    }
+                ));
+            }
+
+            if let Some(ref ident_each) = self.each {
+                tokens.append_all(quote!(
+                    #(#attrs)*
+                    #[allow(unused_mut)]
+                    #vis fn #ident_each <ITEM>(#self_param, item: ITEM) -> #return_ty
```

Comment:
Please use `VALUE` instead of `ITEM` here so that we continue to only reserve one type parameter name for the macro's own use.

Reply:
Done